packages/core/workers/src/Profiler.js

```diff
@@ -40,16 +41,23 @@ export default class Profiler {
   session: Session;
 
   startProfiling() {
-    this.session = new Session();
-    this.session.connect();
+    try {
+      const inspector: any = require('inspector');
+      this.session = new inspector.Session();
+      this.session.connect();
 
-    return Promise.all([
-      this.sendCommand('Profiler.setSamplingInterval', {
-        interval: 100
-      }),
-      this.sendCommand('Profiler.enable'),
-      this.sendCommand('Profiler.start')
-    ]);
+      return Promise.all([
+        this.sendCommand('Profiler.setSamplingInterval', {
+          interval: 100
+        }),
+        this.sendCommand('Profiler.enable'),
+        this.sendCommand('Profiler.start')
+      ]);
+    } catch (err) {
+      throw new ThrowableDiagnostic({
+        diagnostic: errorToDiagnostic(err)
```

Comment:
This would probably result in this error message (which isn't very helpful):
```
🚨  Error: Cannot find module 'inspector'
Require stack:
- .../parcel/packages/core/workers/src/Profiler.js
- ...
```

So instead (to also only catch require errors and not something related to the Session itself): 

```js
let inspector;
try {
    inspector = require('inspectors');
} catch (err) {
    throw new ThrowableDiagnostic({
        diagnostic: {
            message: `The inspector module isn't available`,
            origin: '@parcel/workers',
            hints: ['Disable build profiling']
        }
    });
}

this.session = new inspector.Session();
...
```